### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by sandsebi**

### CODE REVIEW

File: illumina/illumina.nf [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-params.INDEX = "gs://prj-int-dev-covid19-nf-gls/prepro/illumina.index.tsv"
-params.STOREDIR = "gs://prj-int-dev-covid19-nf-gls/prepro/storeDir"
-params.OUTDIR = "gs://prj-int-dev-covid19-nf-gls/prepro/results"
+//TSV file of samples to process
+params.INDEX = "gs://sands-nf-tower/illumina5.tsv"
+params.STOREDIR = "gs://sands-nf-tower/storeDir"
+params.OUTDIR = "gs://sands-nf-tower/results"
+params.CONFIG_YAML = "gs://sands-nf-tower/config.yaml"
+
+//If using LSF, uncomment the following variables and provide an appropriate path to the files
```
Comment (davidyuyuan) [inline illumina/illumina.nf:9]: Is it true that you can use GCS when launching LSF jobs from NXF Tower?

File: README.md [hunk 2]
```diff
@@ -1,39 +1,41 @@
...
-The pipeline supports sequence reads from both Illumina and Nanopore platforms. It is designed to be highly portable for both Google Cloud Platform and High Performance Computing cluster with IBM Spectrum LSF. We have performed secondary and tertiary analysis on millions of public samples. The pipeline shows good performance for large scale production. 
-
-![Component diagram](doc/img/pipeline.components.png)
-
-The pipeline takes SRA from the public FTP from ENA. It submits analysis objects back to ENA on the fly. The intermediate results and logs are stored in the cloud storage buckets or high performance local POSIX file system. The metadata is stored in Google BigQuery for metadata and status tracking and analysis. The runtime is created with Docker / Singularity containers and NextFlow. 
-
-## Process to run the pipelines
-
-The pipeline requires the Nextflow Tower for the application level monitoring. A free test account can be created for evaluation purposes at https://tower.nf/.
+1. Have access to a Google Cloud account
```
Comment (davidyuyuan) [inline README.md:10]: Please add a link to GCP free trial.

File: README.md [hunk 3]
```diff
@@ -1,39 +1,41 @@
...
+2. Have a Nextflow tower account
```
Comment (davidyuyuan) [inline README.md:11]: Please add a link to NXF Tower free trial.

File: README.md [hunk 4]
```diff
@@ -1,39 +1,41 @@
...
-
-### Error handling
-
-If a job is killed or died, run the following to update the metadata to avoid reprocessing samples completed successfully.
-
-1. Run `./covid-sequence-analysis-workflow/update.receipt.sh <batch_id>` to collect the submission receipts and to update submission metadata. The script can be run at anytime. It needs to be run if a batch job is killed instead of completed for any reason.
-2. Run `./covid-sequence-analysis-workflow/set.archived.sh` to update stats for analyses submitted. The script can be run at anytime. It needs to be run at least once before ending a snapshot to make sure that the stats are up-to-date.
-
-To reprocess the samples failed, delete the record in `sra_processing`.
+1. Follow the setup instructions as described on Nextflow tower to setup a [compute environment](https://help.tower.nf/22.4/compute-envs/google-cloud-lifesciences/) on Google
```
Comment (davidyuyuan) [inline README.md:17]: Yanisa wrote a doc when evaluating the NXF Tower. Please check with her where the doc is to see if anything worth mentioning. @mingyanisa ?

File: README.md [hunk 5]
```diff
@@ -1,39 +1,41 @@
...
+2. Follow the setup instructions as described on Nextflow tower to integrate the [codebase](https://help.tower.nf/22.4/git/overview/#github)
+3. Create a project on GCP and a [Bucket](https://console.cloud.google.com/storage/browser) with the following files and folders:
+	a. Folders - data,results,scratch and storeDir
+	b. Files - illumina5.tsv, nanopore5.tsv, data/projects_accounts.csv, [data/NC_045512.2.fa.fai](https://github.com/enasequence/covid-sequence-analysis-workflow/blob/5b6b53b3a228ec55cb1b34bbc3aec5ec31759edd/illumina/illumina.nf#L12), [data/NC_045512.2.fa](https://github.com/enasequence/covid-sequence-analysis-workflow/blob/5b6b53b3a228ec55cb1b34bbc3aec5ec31759edd/illumina/illumina.nf#L11)
```
Comment (davidyuyuan) [inline README.md:21]: 1. It is good that you did not include a copy of data/projects_accounts.csv. Perhaps, it is worth describing what it is about. Perhaps, it is simplier for you and easier for users is to add a dummy file with fake IDs and passwords.
2. Would readers expecting to see links to data/NC_045512.2.fa.fai and data/NC_045512.2.fa files themselves or where they are referenced?

File: README.md [hunk 6]
```diff
@@ -1,39 +1,41 @@
...
-If a job is killed or died, run the following to update the metadata to avoid reprocessing samples completed successfully.
-
-1. Run `./covid-sequence-analysis-workflow/update.receipt.sh <batch_id>` to collect the submission receipts and to update submission metadata. The script can be run at anytime. It needs to be run if a batch job is killed instead of completed for any reason.
-2. Run `./covid-sequence-analysis-workflow/set.archived.sh` to update stats for analyses submitted. The script can be run at anytime. It needs to be run at least once before ending a snapshot to make sure that the stats are up-to-date.
-
-To reprocess the samples failed, delete the record in `sra_processing`.
+1. Follow the setup instructions as described on Nextflow tower to setup a [compute environment](https://help.tower.nf/22.4/compute-envs/google-cloud-lifesciences/) on Google
+2. Follow the setup instructions as described on Nextflow tower to integrate the [codebase](https://help.tower.nf/22.4/git/overview/#github)
+3. Create a project on GCP and a [Bucket](https://console.cloud.google.com/storage/browser) with the following files and folders:
+	a. Folders - data,results,scratch and storeDir
```
Comment (davidyuyuan) [inline README.md:20]: Did you have to results,scratch and storeDir? I believe that they can be created by NXF scripts in this repo.

Yes, you would want to create data folder to add GCS objects in the next step.

File: illumina/illumina.nf (hunk 1 above)
Comment (sandsebi) [reply illumina/illumina.nf:9]: I think we should have a separate section for LSF and for Google. I'll try and add that.

**Pull request closed without merging**